typing/typemod.ml

```diff
@@ -2519,24 +2516,22 @@ and type_structure ?(toplevel = None) funct_body anchor env sstr =
         Tstr_attribute attr, [], env
   in
   let toplevel_sig = Option.value toplevel ~default:[] in
-  let rec type_struct env sstr str_acc sig_acc =
+  let rec type_struct env sstr str_acc sig_acc sig_acc_with_toplvl =
```

Comment:
Maybe there is a better name for this accumulator based on how it is consumed rather than how it is produced?